disperser/cmd/apiserver/main.go

```diff
@@ -114,6 +115,14 @@ func RunDisperserServer(ctx *cli.Context) error {
 		ratelimiter = ratelimit.NewRateLimiter(reg, globalParams, bucketStore, logger)
 	}
 
+	if config.MaxBlobSize <= 0 || config.MaxBlobSize > 64*1024*1024 {
```

Comment:
So we want to support 64MiB?

Reply:
It is just a check, we might experiment with it